app/views/shared/_carousel.html.erb

```diff
@@ -12,7 +12,8 @@
 
     <%# Carousel first slide %>
     <div class="carousel-item active">
-      <a href="/">
+      <% videocassette_one = VideoCassette.find_by(title: "Home Alone") %>
+      <a href="/video_cassettes/<%= videocassette_one.id%>">
```

Comment:
what about using link_to?

Reply:
I need to wrap the entire img with the link, so i dont know who to do that with link_to... maybe you can show me tomorrow if you know how? :)